src/pudl/resources.py

```diff
@@ -0,0 +1,72 @@
+"""Collection of Dagster resources for PUDL."""
+from dagster import Field, resource
+
+from pudl.helpers import EnvVar
+from pudl.settings import DatasetsSettings, FercToSqliteSettings, create_dagster_config
+from pudl.workspace.datastore import Datastore
+
+
+@resource(config_schema=create_dagster_config(DatasetsSettings()))
+def dataset_settings(init_context):
+    """Dagster resource for parameterizing assets.
+
+    This resource allows us to specify the years we want to process for each datasource
+    in the Dagit UI.
+
+    We configure the assets using a resource instead of op configs so the settings can
+    be accesible by any op.
+    """
+    return DatasetsSettings(**init_context.resource_config)
+
+
+@resource(config_schema=create_dagster_config(FercToSqliteSettings()))
+def ferc_to_sqlite_settings(init_context):
+    """Dagster resource for parameterizing assets.
+
+    This resource allows us to specify the years we want to process for each datasource
+    in the Dagit UI.
+
+    We configure the assets using a resource instead of op configs so the settings can
+    be accesible by any op.
+    """
+    return FercToSqliteSettings(**init_context.resource_config)
+
+
+@resource(
+    config_schema={
+        "local_cache_path": Field(
+            EnvVar(
+                env_var="PUDL_CACHE",
+            ),
+            description="Path to local cache of raw data.",
+            default_value=None,
+        ),
+        "gcs_cache_path": Field(
```

Comment:
So in theory I could configure my dag run to use `gcs_cache_path` by chucking that into the `resources` map in a run configuration like
```
resources:
  gcs_cache_path: gs://path-to-bucket-that-i-forget
  dataset_settings:
    -yada yada
```
?

Reply:
Yup!